Wishlist/wishlist.css

```diff
@@ -0,0 +1,60 @@
+@import url(/Utility.css);
+
+
+html,
+body {
+  max-width: 100%;
+  overflow-x: clip;
+}
+
+hr {
+  margin: 0;
+}
+.grid-container {
+  display: grid;
+  grid-template-columns: auto 1fr 1fr 1fr 1fr 1fr;
+  grid-template-areas:
+    "navbar navbar navbar navbar navbar navbar"
+    "header header header header header header"
+    "aside main main main main main"
+    "aside main main main main main"
+    "aside main main main main main"
+    "footer footer footer footer footer footer";
+}
+
+.navbar {
+  grid-area: navbar;
+  min-height: 11vh;
+  background-color: var(--pure-white);
+  box-shadow: var(--box-shadow-dark);
+  padding: var(--value-2) var(--value-2);
+  top: 0;
+  z-index: 10;
+  width: 100vw;
```

Comment:
why 10 z-index ?

Reply:
Ohh, I have set my highest z-index as 10, which is the navbar. Other z-index values are between 0 to 10.